frontend/app/styles/partials/create-badge.scss

```diff
@@ -31,6 +31,3 @@
   }
 }
 
-.hide {
-  display: none !important;
```

Comment:
I would be happy if this file was only rolled back.

Reply:
I have rolled back this.

Reply:
why is this deleted?